assets/js/modules/analytics/setup/setup-form.js

```diff
@@ -60,14 +60,14 @@ export default function SetupForm( { finishSetup } ) {
 		>
 			<ErrorNotice />
 
+			<ExistingTagNotice />
+
 			{ ( !! accounts.length && ! hasExistingTag ) && (
-				<p>
+				<p style={ { marginBottom: 0 } }>
```

Comment:
😱

Any chance we can get a CSS class for this? I know it's just in one place but feels like a dangerous precedent to set.

Reply:
It isn't the first inline style in a component, but sure I'll add a utility 👍